interface/wx/event.h

```diff
@@ -1741,10 +1741,17 @@ class wxJoystickEvent : public wxEvent
     /**
         Returns the identifier of the button changing state.
 
-        This is a @c wxJOY_BUTTONn identifier, where @c n is one of 1, 2, 3, 4.
+        This was once a @c wxJOY_BUTTONn identifier, where @c n is one of 1, 2, 3, 4,
```

Comment:
Sorry, this seems to be a bit unclear to me. What about:

```suggestion
        The return value is @code 1 << n @endcode where @c n is the index of the
        button changing state.
        
        Note that for @c n equal to 1, 2, 3 or 4 there are predefined @c wxJOY_BUTTONn
        constants which can be used for more clarity, however these constants are not
        defined for the buttons beyond the first four.
```